### PR TITLE
chore: document batch message content endpoint

### PR DESCRIPTION
**Pull request by cellomatt**
### Description

This PR documents the new `GET /v1/messages/batch/content` endpoint added in [#2511](https://github.com/knocklabs/switchboard/pull/2511).

https://docs-15ps0u3mn-knocklabs.vercel.app/reference#batch-get-message-content

### CODE REVIEW

File: content/reference.mdx [hunk 1]
```diff
@@ -3212,6 +3213,65 @@ of message being sent (email, chat, in-app feed, sms and push).**
...
+      },
+      "html_body": "<p>example</p>",
+      "reply_to": "reply-to@example.com",
+      "subject": "Example Notification",
+      "text_body": "example",
+      "to": "user@example.com"
+    },
+    "inserted_at": "2021-04-06T12:00:00Z"
+  }
+]
```
Comment (rachael-t) [inline content/reference.mdx:3269]: Small nit, but I think it would be helpful for this example response to display more than one object

**Review by rachael-t — APPROVED** [review]
Thank you for doing this! 🙏

File: content/reference.mdx (hunk 1 above)
Comment (cellomatt) [reply content/reference.mdx:3269]: Fair, I was worried it would be too long but I can do that!

**Review by cjbell — APPROVED** [review]
💯

**Pull request merged**